src/cst1201/Simulator.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cst1201;
+
+/**
+ *
+ * @author Yadadye
+ */
+public class Simulator {
+
+    public static void main(String[] args) {
+       InitialDrawing myBowTie = new InitialDrawing();

```

Comment:
`myBowTie` is probably not a good variable name for a reference variable referring to an `InitialDrawing` object. Probably something like `myInitialDrawing` would be better.